code/src/components/AddTodo.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+
+import uniqid from 'uniqid'
+import styled from "styled-components";
+
+import { todos } from '../reducers/todos'
+
+
+export const AddTodo = () => {
+    const [inputValue, setInputValue] = useState(''); 
+    const dispatch = useDispatch();
+ 
+    const onItemAdd = (event) => {
+        event.preventDefault();
+        dispatch(
+            todos.actions.addTodoItem({
+                newTodo: {
```

Comment:
Another thing you should check in your Prettier's settings is to set indentation for 2 spaces by default.